tests/test_securityhub.py

```diff
@@ -270,19 +269,7 @@ def test_instance_findings_filter_fail_validate(self):
             session_factory=factory,
             validate=False
         )
-        self.assertRaises(PolicyValidationError, policy.validate())
```

Comment:
the deprecation warning and bug here is that the validate method should be invoked by assert raises, instead its passing the non callable return value here. with the change to use the security hub sdk shape for validation though, this will no longer raise an exception and there really isn't much point to the test as a whole then, given its written to expect an error here, the whole test should probably be removed.